### DISCUSSION TITLE
🧩 FEATURE REQUEST: Daily bot news for Telegram

### DISCUSSION THREAD

**Discussion by wtrungvu** [dailydotdev/daily#934] [Feature Requests]
### Concise Description

Daily bot news for Telegram

### Description

Create a Telegram channel, users click the follow button and receive daily news via Telegram notifications.

### Solution Proposed

Create a Telegram channel, users click the follow button and receive daily news via Telegram notifications.

### Code of Conduct

- [x] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @wtrungvu,

Thanks for the suggestion! What functionality would you expect from such bot?

**Reply by wtrungvu** [reply]
Here are some ideas and functionalities you might consider:
1. Daily News Updates: The bot should fetch and deliver newsworthy content on a daily basis. This could include news articles, blog posts, or summaries of trending topics.
2. Customizable Topics: Allow users to select their preferred news topics to receive personalized updates. This could include categories like Java, DevOps, Cloud, etc.
3. Follow/Unfollow Feature: Implement a "Follow" button so that users can choose to subscribe to the channel and start receiving daily news notifications. Provide an option to unsubscribe or "Unfollow" if they no longer wish to receive updates.
4. Notification Settings: Allow users to customize their notification preferences, such as the frequency of notifications (e.g., once per day or multiple times), specific time of day, or even the format of notifications (e.g., short summaries or full articles).
5. Read More: Include a "Read More" button or link with each notification, redirecting users to the full news article or content for further reading.
6. Trending News: Highlight popular or trending news topics to grab users' attention and keep them informed about the most relevant stories.
7. Feedback and Suggestions chat.
8. General discussion chat.
9. Upcoming tech events, live stream notifications, video, etc.

**Reply by idoshamun** [reply]
Thanks a lot!

**Comment by capJavert** [comment]
**Update: daily.dev now has a public API** 🎉

This one's been asked for here for a long time, so: it's live, and it's
available to everyone. Generate a personal access token at
[daily.dev/settings/api](https://daily.dev/settings/api) and start calling it.

What's in v1:

- **Feeds** — your personalized For You feed, trending posts, filtering by tag or source
- **Posts** — full post details, AI summaries and comment threads
- **Search** — posts, tags and sources
- **Bookmarks** — read and manage your saved content
- **Notifications** — your activity feed and unread counts
- **Profile and reading streak** — your experience, tech stack and streak

📣 The announcement: https://daily.dev/posts/public-api-access-is-now-open-for-everyone-ljj8kh9sa
📚 Docs: https://docs.daily.dev/public-api/
